styles/google-gemini/catppuccin.user.css

```
@@ -232,9 +246,24 @@
 
     --og-theme-color: @text;
 
+    span {
+        color: @text;
+    }
+
+    .new-feature {
+        background: @accent-color;
+        color: @base;
+    }
```

Why is it @accent-color. surface or overlay will be more appropriate here

```suggestion
    .new-feature {
        background: @surface2;
    }
```

---

i thought ``@accent-color`` was appropriate for this